service/dispatch/arch-go.yaml

```diff
@@ -0,0 +1,96 @@
+version: 1
+threshold:
+  compliance: 100
+  coverage: 40
+dependenciesRules:
+  - package: "**.internal/clients.*"
+    shouldOnlyDependsOn:
```

Comment:
Isn't it a bit extra? Looks like you collect the cons from both approaches. Moreover, it is quite obvious that packages do not depend on mocks and tests. What do you think about this?

Reply:
Yeah, it's a bit extra (my inner chekist desires to suppress any chance of rebellion 😅).
I've removed usage of shouldNotDependsOn.

Reply:
I can't stop laughing ))